src/clusterfuzz/_internal/bot/fuzzers/honggfuzz/engine.py

```diff
@@ -157,8 +167,13 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
         target_path,
     ])
 
+    honggfuzz_env = {}
+    if _contains_netdriver(target_path):
+      honggfuzz_env['HFND_TCP_PORT'] = '8000'
```

Comment:
Not 100% sure. I assume it might depend on how the fuzz driver is implemented as the fuzzer itself won't start since it will try to listen to the port but be rejected. 

The port is tied to the compiled binary, i.e. it's specified in the fuzzer e.g. https://github.com/google/oss-fuzz/pull/7351/files#diff-78f7a0745bb850a0ce8d42a7c0928d6dc20dda6d39b94b86ea8c60341d48f130R26 so I don't think it's easy to make this flexible and we should rather define a fixed port to be used.

I suggest we bail out here if the port is in use.

WDYT?

Reply:
I think we don't need to implement special handling. Lets let honggfuzz fail.
But I think it would be better to use something like 8666 that is less common.

Reply:
sounds good! done